doc/api/buffer.md

```diff
@@ -869,6 +869,10 @@ The index operator `[index]` can be used to get and set the octet at position
 `index` in `buf`. The values refer to individual bytes, so the legal value
 range is between `0x00` and `0xFF` (hex) or `0` and `255` (decimal).
 
+Since this operator is inherited from `Uint8Array`, its behavior on
+out-of-bounds access is exactly the same - i.e. getting returns `undefined`,
+setting does nothing.
+
```

Comment:
That seems more wordy than necessary, and I'd have to repeat the same thing about setting. Also it would have to be "greater or equal".

>I'm not sure there's much value is calling out the inheritance from Uint8Array

There are finer nuances in its behavior, for example how [indexed access on a Buffer always ignores the prototype chain](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#Property_access). Referencing `Uint8Array` means people know where to read about them if they're interested.